draft-ietf-jsonpath-base.md

```diff
@@ -416,17 +417,17 @@ be used with a comparable XML document; see also {{xpath-overview}}.
 ## Overview {#synsem-overview}
 
 A JSONPath query is a string which selects zero or more nodes of a piece of JSON.
-A valid query conforms to the ABNF syntax defined by this document.
 
-A query MUST be encoded using UTF-8. To parse a query according to
-the grammar in this document, its UTF-8 form SHOULD first be decoded into
+A query MUST be encoded using UTF-8.
+The grammar for queries given in this document assumes that its UTF-8 form is first be decoded into
 Unicode code points as described
-in {{RFC3629}}.
+in {{RFC3629}}; implementation approaches that lead to an equivalent
+result are possible.
```

Comment:
The statement after the semicolon seems unnecessary. Isn't that always the case for IETF or any other specifications?

Reply:
Yes, but we also tend to always point that out.